src/core/UBSceneCache.cpp

```diff
@@ -49,7 +49,8 @@ UBSceneCache::UBSceneCache()
 
 UBSceneCache::~UBSceneCache()
 {
-    // NOOP
+    // delete all cached scenes
```

Comment:
This provokes a sporadic crash during shutdown. To be analyzed under which circumstances a scene would be deleted twice.

Reply:
Fixed. Removed the change on the Scene Cache, which was probably not the right thing to do. Changing ownership of web views was sufficient.